### PR TITLE
Fix randomly failing tests

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: Tests/Routing/DynamicRouterTest.php [hunk 1]
```diff
@@ -147,9 +147,12 @@ public function testMatchUrl()
         ;
 
         $expected = array('this' => 'that');
+        $test = $this;
         $this->enhancer->expects($this->once())
             ->method('enhance')
-            ->with($this->equalTo($routeDefaults), $this->equalTo($this->request))
+            ->with($this->equalTo($routeDefaults), $this->callback(function (Request $request) use ($test) {
```
Comment (ElectricMaxxx) [inline Tests/Routing/DynamicRouterTest.php]: There is some php closure fuckup in v5.3 here. Can't we pass the url directly?

Comment (WouterJ) [inline Tests/Routing/DynamicRouterTest.php]: fix it

Comment (ElectricMaxxx) [inline Tests/Routing/DynamicRouterTest.php]: uhh even better :-)

**Comment by dbu** [comment]
thanks wouter!
what was it? timestamp issues with request objects?

**Comment by WouterJ** [comment]
Yeah, but I'm not really sure why the timestamp changed (as why seem to used the same object instance twice).

**Pull request merged**